inc/admin/dashboard/namespace.php

```diff
@@ -23,6 +23,30 @@ function get_rss_defaults() {
 	];
 }
 
+/**
+ * @param array $options
+ * @return bool
+ */
+function should_display_custom_feed( array $options ): bool {
+	if ( ! $options['display_feed'] ) {
+		return false;
+	}
+
+	$plugins_config_is_active = is_plugin_active( 'pressbooks-plugins-config/pressbooks-plugins-config.php' );
```

Comment:
@fdalcin I think pressbooks core should not know that pressbooks-plugin-config exists I suggest to `hookify` this instead doing that check probably you could create an action hook in pb-config and check for the existance in pressbooks core

**pressbooks-plugin-config**
```
add_filter(
				'should_display_custom_feed', function() {
					return true;
				}
			);
```

**pressbooks**
```
if(has_filter( 'should_display_custom_feed' )) {

}
```

What do you think?

Reply:
I like the idea! Sent changes here and in pressbooks/pressbooks-plugins-config#16. Let me know what you think and if you have any suggestion